sensing/src/main/java/com/google/android/sensing/upload/AssumeRoleWithCustomTokenBasedCredentialProvider.kt

```diff
@@ -33,32 +33,34 @@ import org.simpleframework.xml.Path
 import org.simpleframework.xml.Root
 
 /**
- * Credential Provider using AssumeRoleWithCustomToken API. Extends and implements
- * AssumeRoleBaseProvider.
+ * Implementation of [io.minio.credentials.Provider] using AssumeRoleWithCustomToken API. Extends
+ * and implements [io.minio.credentials.AssumeRoleBaseProvider].
  */
-class CustomTokenIdentityProvider(
+class AssumeRoleWithCustomTokenBasedCredentialProvider(
```

Comment:
nit: The class name seems to be big, the earlier one seemed fine to me. Any reasons why this is changed?

Reply:
done.